configs/HGLRCH743/config.h

```diff
@@ -0,0 +1,136 @@
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_MAX7456
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+#define SPI1_SCK_PIN            PA5
+#define SPI1_SDI_PIN            PA6
+#define SPI1_SDO_PIN            PA7
+#define GYRO_1_EXTI_PIN         PC5
+#define GYRO_1_CS_PIN           PA4
+#define GYRO_1_SPI_INSTANCE     SPI1
+
+#define SPI2_SCK_PIN            PB13
+#define SPI2_SDI_PIN            PB14
+#define SPI2_SDO_PIN            PB15
+#define MAX7456_SPI_CS_PIN      PB12
+#define MAX7456_SPI_INSTANCE    SPI2
+
+#define SPI3_SCK_PIN            PB3
+#define SPI3_SDI_PIN            PB4
+#define SPI3_SDO_PIN            PB2
+#define FLASH_CS_PIN 						PA15
+#define FLASH_SPI_INSTANCE      SPI3
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+
+#define BEEPER_PIN              PE5
+#define MOTOR1_PIN              PE9
+#define MOTOR2_PIN              PE11
+#define MOTOR3_PIN              PE13
+#define MOTOR4_PIN              PE14
+#define MOTOR5_PIN              PA2
+#define MOTOR6_PIN              PA3
+#define MOTOR7_PIN              PB0
+#define MOTOR8_PIN              PB1
+#define LED_STRIP_PIN           PA0
+#define SERVO1_PIN              PD12
+#define SERVO2_PIN              PD13
+#define SERVO3_PIN              PD14
+#define SERVO4_PIN              PD15
+
+#define UART1_TX_PIN            PA9
+#define UART1_RX_PIN            PA10
+#define UART2_TX_PIN            PD5
+#define UART2_RX_PIN            PD6
+#define UART3_TX_PIN            PD8
+#define UART3_RX_PIN            PD9
+#define UART4_TX_PIN            PD1
+#define UART4_RX_PIN            PA1
+#define UART5_TX_PIN            PB6
+#define UART5_RX_PIN            PB5
+#define UART6_TX_PIN            PC6
+#define UART6_RX_PIN            PC7
+#define UART7_TX_PIN            PE8
+#define UART7_RX_PIN            PE7
+#define UART8_TX_PIN            PE1
+#define UART8_RX_PIN            PE0
+
+#define I2C1_SCL_PIN            PB8
+#define I2C1_SDA_PIN            PB7
+#define I2C2_SCL_PIN            PB10
+#define I2C2_SDA_PIN            PB11
+
+#define LED0_PIN                PE6
+#define LED1_PIN                PE4
+#define LED2_PIN                PE3
+
+#define ADC_VBAT_PIN            PC0
+#define ADC_CURR_PIN            PC1
+#define ADC_RSSI_PIN            PC4
+
+#define PINIO1_PIN              PE2
+#define PINIO2_PIN              PC13
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(0,  PE9,   1,  4) \
+    TIMER_PIN_MAP(1,  PE11,  1,  5) \
+    TIMER_PIN_MAP(2,  PE13,  1,  6) \
+    TIMER_PIN_MAP(3,  PE14,  1,  7) \
+    TIMER_PIN_MAP(4,  PA2,   2,  0) \
+    TIMER_PIN_MAP(5,  PA3,   2,  0) \
+    TIMER_PIN_MAP(6,  PB0,   2,  0) \
+    TIMER_PIN_MAP(7,  PB1,   2,  0) \
+    TIMER_PIN_MAP(8,  PD12,  1, -1) \
+    TIMER_PIN_MAP(9,  PD13,  1,  0) \
+    TIMER_PIN_MAP(10, PD14,  1, -1) \
+    TIMER_PIN_MAP(11, PD15,  1, 14) \
+    TIMER_PIN_MAP(12, PA0,   1,  0) 
```

Comment:
Incorrect DMA opt for pin, is overwriting previous "0" DMA opt assignment.

```suggestion
    TIMER_PIN_MAP(12, PA0,   1,  10) 
```